tests/e2e/specifications/input_data.yaml

```diff
@@ -1 +1,2 @@
-file: /mnt/team/simulation_science/priv/engineering/er_ecosystem/sample_data/dummy/input_file_1.parquet
+# file: /mnt/team/simulation_science/priv/engineering/er_ecosystem/sample_data/dummy/input_file_1.parquet
+file: /mnt/share/homes/sbachmei/venv/input_data/input_file_1.parquet
```

Comment:
This shouldn't refer to your personal directory

Reply:
Oof, good catch. I did that a _long_ time ago before we set up the service user.